<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>871</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background :rgb(255, 229, 237) ;border-radius : 10;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>961</width>
     <height>761</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>961</width>
     <height>761</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background :qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 246, 254, 255), stop:1 rgba(255, 255, 255, 255));border-radius : 20;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save_bt">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(102, 112, 98);
border-radius : 10px;
color:white}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ex_bt">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>700</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(102, 112, 98);
border-radius : 10px;
color:white}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_bt">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(102, 112, 98);
border-radius : 10px;
color:white}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tw_ed">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color:rgb(255, 255, 255);border-radius : 6px;
color:black}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hastag Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background :rgb(255, 219, 229);border-radius : 20;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>891</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : rgb(190, 210, 255) ;border-radius : 10;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>bg_3</zorder>
  <zorder>bg</zorder>
  <zorder>bg_2</zorder>
  <zorder>tableView</zorder>
  <zorder>save_bt</zorder>
  <zorder>ex_bt</zorder>
  <zorder>search_bt</zorder>
  <zorder>tw_ed</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
